From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 5 Aug 2017 02:03:01 +0200
Subject: [PATCH] maas: region: skip credentials update

Password update for maas psql database breaks ulterior acesses
to maas-region syncdb.
For now, limit regiond.conf changes to maas_url, and skip
updating credentials.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---

diff --git a/maas/region.sls b/maas/region.sls
index d3227ca..8a2243d 100644
--- a/maas/region.sls
+++ b/maas/region.sls
@@ -6,10 +6,9 @@
     - names: {{ region.pkgs }}

 /etc/maas/regiond.conf:
-  file.managed:
-  - source: salt://maas/files/regiond.conf
-  - template: jinja
-  - group: maas
+  file.replace:
+  - pattern: ^maas_url.*$
+  - repl: "maas_url: http://{{ region.bind.host }}/MAAS"
   - require:
     - pkg: maas_region_packages

